scripts/docker_build.sh

```diff
@@ -20,7 +20,7 @@
 DOCKER_ARGS=$@
 
 GIT_ROOT=$(git rev-parse --show-toplevel)
-GIT_REVISION=$($GIT_ROOT/gradlew -q properties --property version | grep -oP "version: \K.+")
+GIT_REVISION=$("$GIT_ROOT/gradlew" -q properties --property version | grep "version" | sed -e "s/version: //")
```

Comment:
Some things, please
- move the addition of quotes to a separate commit, which a separate rationale,
- explain what part of the `grep`command wasn't working on macOS,
- use either `grep` or `sed`, but not both.

Reply:
Fixed

Reply:
There's now a merge commit in the PR.